src/cli.mjs

```diff
@@ -104,6 +104,18 @@ function buildYargs(argvInput) {
     .env("FAUNA")
     .config("config", configParser)
     .middleware([checkForUpdates, logArgv], true)
+    .middleware((argv) => {
+      if (!argv.url) {
+        if (argv.local) {
+          argv.url = 'http://localhost:8443';
+        } else {
+          argv.url = 'https://db.fauna.com';
+        }
+      }
+      if (!argv.secret && argv.local) {
+        argv.secret = "secret";
+      }
+    })
```

Comment:
If you move this into a function, we can unit test it as an individual middleware.

Reply:
will do. i love my end to end tests tho.